Automattic/wp-calypso
# Stats: Subscriber Stats add mock data

This is just a draft PR suggestion for how we might integrate mock data 

Related to https://github.com/Automattic/wp-calypso/issues/74403

## Proposed Changes

* Creates a separate mock data store in Redux (client/state/stats) to use temporarily until real data is available.
* Uses `connect()` function along with a `mapStateToProps()` function to map the mocked data object to the component's props and use as needed in our new section.
* Creates a new `StatsSubscribers` section
* Once the real data is ready, we can replace the connect() function and remove the mock data store in redux.

## Testing Instructions

* Open the Calypso Live Branch
* On the Stats dashboard, check that there is data being output (just in text form at this stage, and not in the right location on the page. The purpose of this PR is the location and fetching of the mock data) 


## Pre-merge Checklist
